nflow-engine/src/main/java/io/nflow/engine/internal/dao/WorkflowInstanceDao.java

```diff
@@ -99,8 +98,8 @@
   private long workflowInstanceQueryMaxResultsDefault;
   private long workflowInstanceQueryMaxActions;
   private long workflowInstanceQueryMaxActionsDefault;
-  int instanceStateTextLength;
-  int actionStateTextLength;
+  int instanceStateTextLength = -1;
```

Comment:
No, sorry WorkflowInstanceDaoTest.java sets them manually, so that's why they are package private. I'll not change that, if that's ok.